CMakeLists.txt

```diff
@@ -715,16 +710,16 @@ add_library(MINIZIP::minizip ALIAS ${PROJECT_NAME})
 # Install files
 if(NOT SKIP_INSTALL_LIBRARIES AND NOT SKIP_INSTALL_ALL)
     target_include_directories(${PROJECT_NAME} PUBLIC
-        $<INSTALL_INTERFACE:${INSTALL_INC_DIR}>)
+        $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>)
 
     install(TARGETS ${PROJECT_NAME} ${MINIZIP_DEP}
             EXPORT ${PROJECT_NAME}
-            INCLUDES DESTINATION "${INSTALL_INC_DIR}"
-            RUNTIME DESTINATION "${INSTALL_BIN_DIR}"
-            ARCHIVE DESTINATION "${INSTALL_LIB_DIR}"
-            LIBRARY DESTINATION "${INSTALL_LIB_DIR}")
+            INCLUDES DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/minizip-ng"
```

Comment:
Includes DIR should be different if using compat mode?

Reply:
OK I've added commit to insytall headers in${CMAKE_INSTALL_INCLUDEDIR}/minizip-ng.
It was necessary update as well .pc file.